packages/fxa-auth-server/lib/routes/account.js

```diff
@@ -1512,19 +1513,40 @@ module.exports = (
 
         const { uid, scope } = request.auth.credentials;
         const { ecosystemAnonId } = request.payload;
-        const noneMatchHeader = request.headers['If-None-Match'];
         const scopeSet = ScopeSet.fromArray(scope);
 
+        function hashAnonId(anonId) {
+          const hash = crypto.createHmac(
```

Comment:
No need for mac, just hash is preferable https://nodejs.org/dist/latest-v12.x/docs/api/crypto.html#crypto_class_hash